pmm-app/tests/pages/qanPage.js

```diff
@@ -328,4 +335,61 @@ module.exports = {
     I.waitInUrl('tz=browser&theme=dark', 30);
     I.seeInCurrentUrl(urlPart);
   },
+
+  waitForNewQANPageLoaded() {
+    I.waitForElement(this.fields.newQANPanelContent, 30);
+    I.waitForInvisible(this.fields.newQANSpinnerLocator, 30);
+  },
+
+  applyFilterNewQAN(filterName) {
+    const filterToAplly = `//span//span[contains(@class, 'checkbox-container__label-text') and contains(text(), '${filterName}')]`;
```

Comment:
we don't really need two `//span//span` I think

Reply:
You're right! :)